src/chocolatey/infrastructure/licensing/License.cs

```diff
@@ -50,22 +50,15 @@ public static ChocolateyLicense validate_license()
                 catch (Exception ex)
                 {
                     "chocolatey".Log().Error(
-@"Error when attempting to load chocolatey licensed assembly. Ensure
- that chocolatey.licensed.dll exists at
- '{0}'.
- The error message itself may be helpful:{1} {2}".format_with(
-                    ApplicationParameters.LicensedAssemblyLocation,
-                    Environment.NewLine,
-                    ex.Message
-                    ));
-                    "chocolatey".Log().Warn(ChocolateyLoggers.Important,@" Install the Chocolatey Licensed Extension package with
- `choco install chocolatey.extension` to remove this license warning.
- TRIALS: If you have a trial license, you cannot use the above command
- as is and be successful. You need to download nupkgs from the links in
- the trial email as your license will not be registered on the licensed
- repository. Please reference
- https://docs.chocolatey.org/en-us/licensed-extension/setup#how-do-i-install-the-trial-edition
- for specific instructions.");
+                        @"A valid Chocolatey license was found, but the chocolatey.licensed.dll assembly could not be loaded:
+  {0}
+Ensure that the chocolatey.licensed.dll exists at the following path:
+ '{1}'".format_with(ex.Message, ApplicationParameters.LicensedAssemblyLocation));

```

Comment:
We're not passing the error message in ` ex.Message` through here? Is there a reason for this?

Reply:
I'm not sure what you mean? The `format_with()` will insert the value of `ex.Message` into the format string, which is then logged as an error.

Reply:
DOH! I genuinely didn't see the `ex.Message` in yours. It's right at the beginning 😄